lib/bgsite_official_web/templates/tag/show.html.leex

```diff
@@ -38,8 +44,13 @@
        <br>
      </div>
    </div>
- <% end %>
 
+ <% end %>
 
  </div>
 </div>
+
+<%= def handle_event("like",_,socket) do
+    BgsiteOfficial.WebsiteLive.inc_likes(socket.assings.post)
+    {:noreply,socket}
+    end %>
```

Comment:
This is totally wrong, Elixir Phoenix is not PHP we dont do that here. Its a `html.leex` template, not controller /live view/